.circleci/src/jobs/@web-jobs.yml

```diff
@@ -270,17 +270,17 @@ web-deploy-sentry-sourcemaps:
         name: cut-sentry-release
         command: |
           cd packages/web
-          ../../node_modules/.bin/sentry-cli --auth-token ${SENTRY_AUTH_TOKEN} releases --org audius --project audius-client new ${CIRCLE_SHA1}
+          npm run sentry-cli -- --auth-token ${SENTRY_AUTH_TOKEN} releases --org audius --project audius-client new ${CIRCLE_SHA1}
     - run:
         name: upload-sourcemaps
         command: |
           cd packages/web
-          ../../node_modules/.bin/sentry-cli --auth-token ${SENTRY_AUTH_TOKEN} releases --org audius --project audius-client files ${CIRCLE_SHA1} upload-sourcemaps --no-rewrite build-production/static/js
+          npm run sentry-cli -- --auth-token ${SENTRY_AUTH_TOKEN} releases --org audius --project audius-client files ${CIRCLE_SHA1} upload-sourcemaps --no-rewrite build-production/static/js
```

Comment:
fyi I think you can also just do `npx --no-install sentry-cli` and we don't need the script in package.json